.changeset/eleven-adults-add.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes-backend': minor
```

Comment:
if I'm not mistaken this can be a `patch` as it's not a breaking change & we are still on a `0.x.y` version with this plugin :) 

```suggestion
'@backstage/plugin-kubernetes-backend': patch
```